static/src/stylesheets/module/email-signup/_form.scss

```diff
@@ -199,6 +213,12 @@
     margin: 0 0 $gs-baseline;
 }
 
+.email-sub__message__headline__footer,
+.email-sub__message__description__footer {
```

Comment:
This reads as if you're targeting the headline footer, and description footer, which I think you're targeting the email subscription footer version. 

This is BEM which is block, element, modifier, technically this CSS class should be: 

`email-sub-footer__message__headline`

Reply:
Thanks for this. Didn't realise there was a convention for naming so it's good to know moving forwards!